### ISSUE TITLE
javy not appearing on build status page

### ISSUE THREAD

**Issue by jeffcharles** [google/oss-fuzz#12792]
👋 [this PR](https://github.com/google/oss-fuzz/pull/12754) to integrate Javy into OSS-Fuzz was merged on Monday Nov 25 but I'm not seeing javy appearing under any of the _Fuzzing Builds_, _Coverage Builds_, or _Introspector Builds_ on https://oss-fuzz-build-logs.storage.googleapis.com/index.html. I do see Javy listed as a project when visiting https://oss-fuzz.com/.

Was there something I missed in the integration PR that would result in the builds not happening? Or otherwise not appearing on that status page?

**Comment by jeffcharles** [comment]
Yes, I can see Javy listed under _Fuzzing Builds_ now. I still don't see a listing under _Coverage Builds_ or _Introspector Builds_ however.

**Comment by DavidKorczynski** [comment]
> Yes, I can see Javy listed under _Fuzzing Builds_ now. I still don't see a listing under _Coverage Builds_ or _Introspector Builds_ however.

My guess would be that it's because of https://github.com/google/oss-fuzz/blob/06a694ed8ee65428f29d220a001681a19ccde657/projects/javy/project.yaml#L7-L8
